packages/web-components/src/global/_formation_overrides.scss

```diff
@@ -118,3 +118,23 @@
 .usa-select {
   background-size: rem-override(1.25rem);
 }
+
+.usa-memorable-date .usa-form-group {
+  margin-top: 0px;
+}
+
+.usa-form-group--day, .usa-form-group--month, .usa-form-group--year {
+  margin-right: rem-override(1rem);
+}
+
+.usa-form-group--day-input, .usa-form-group--month-input {
+  width: rem-override(3rem);
+}
+
+.usa-form-group--year-input {
+  width: rem-override(4.5rem);
+}
+
+.usa-form-group--month-select {
+  width: rem-override(15rem);
+}
```

Comment:
You are still missing the pixel conversion for `.usa-select { background-position: right 0.5rem center} `

<img width="917" alt="Screenshot 2023-03-08 at 4 23 24 PM" src="https://user-images.githubusercontent.com/872479/223865012-968f77d0-c435-476f-b0fb-ddd6403e99c4.png">